css/css-color/predefined-014.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 4: predefined colorspaces, fallback</title>
+<link rel="author" title="Chris Lilley" href="mailto:chris@w3.org">
+<link rel="help" href="https://drafts.csswg.org/css-color-4/#predefined">
+<link rel="match" href="greensquare-ref.html">
+<meta name="assert" content="Color function with invalid colorspace, fallback matches sRGB #008000">
+<style>
+    .test { background-color: red; width: 12em; height: 6em; margin-top:0}
+    .ref { background-color: #008000; width: 12em; height: 6em; margin-bottom: 0}
+    .test {background-color: color(banana 0.1 0.2 0.3, #008000)}

```

Comment:
Seems like it might be worth throwing in a test with red fallback after a valid value, also.

Reply:
Good point. Added as predefined-015.html